README.md

```diff
@@ -99,6 +99,10 @@ Another useful tool for making certificates is [mkcert].
 
 [mkcert]: https://github.com/FiloSottile/mkcert
 
+## MSRV Policy
+
+The Minimum Supported Rust Version (MSRV) of this crate is **1.47.0**. This crate is guaranteed to build on any version of Rust at or after the MSRV. The MSRV will not be changed without at least a minor bump in this crate's version. As a policy, the MSRV will not advance past the [current Rust version provided by Debian Stable](https://packages.debian.org/stable/rust/rustc). At the time of writing, this version of Rust is *1.48.0*. However, the MSRV may be advanced further in the event of a major ecosystem shift or a security vulnerability.
```

Comment:
> This crate is guaranteed to build on any version of Rust at or after the MSRV. The MSRV will not be changed without at least a minor bump in this crate's version.

I think it is very difficult to guarantee this.

- For the later, it is often automatically bumped by dependencies (including other smol-rs crates).
- For the former, I think the best we can actually guarantee is "guarantees that at least it can be built with the latest dependencies available at the time that version was published." (I sometimes see people expecting a bigger guarantee than this.)

Personally, I would suggest weakening or removing the guarantee for now.

> As a policy, the MSRV will not advance past the [current Rust version provided by Debian Stable](https://packages.debian.org/stable/rust/rustc).

nit: I would prefer to state that this is "a tentative policy". Like regex: https://github.com/rust-lang/regex#minimum-rust-version-policy

Also, I think smol-rs/smol and its dependencies are fine with this (tentative) policy, but we may want to avoid setting the policy on other smol-rs crates for now. (see also https://github.com/smol-rs/smol/issues/244#issuecomment-1366737852)

Reply:
I meant to suggest removing "The MSRV will not be changed without at least a minor bump in this crate's version" too.